### ISSUE TITLE
FreeRDP client quote marks in user name

### ISSUE THREAD

**Issue by krzychoo23** [Thinstation/thinstation#664]
FreeRDP client add quote marks to user name after pressing Reconnect.
My FreeRDP config in the inthinstation.conf is:

SESSION_0_TYPE=freerdp
SESSION_0_TITLE="RDP Terminal Server"
SESSION_0_FREERDP_SERVER=myhost.domain.com
SESSION_0_FREERDP_OPTIONS='/cert-ignore /u:'

SESSION_0_AUTOSTART=on
RECONNECT_PROMPT=MENU
NO_SESSION=REBOOT

When I type correct user name and pass FreeRDP connects to Windows RDP server. But when I intentionally or not intentionally type wrong pass FreeRDP asks for Reconnect or Restart (it is ok). After pressing Reconnect user name has extra quotes e.g. 'myusername' instead myusername. So, I have to click and remove extra marks from user field to establish connection again.
How to configure FreeRDP Client NOT to add quote marks?

regards, krzychoo23

**Comment by Thinstation** [comment]
I made a bad commit over the weekend. Might be fixed now.

**Comment by krzychoo23** [comment]
Did you release new update to DevStation? I did "Update DevStation" but "No update Needed". I builded client again anyway but i have still quotes in user name after reconnect.

**Comment by Doncuppjr** [comment]
Factory reset


Sent from Yahoo Mail for iPhone


On Monday, August 30, 2021, 11:45 PM, krzychoo23 ***@***.***> wrote:




Did you release new update to DevStation? I did "Update DevStation" but "No update Needed". I builded client again anyway but i have still quotes in user name after reconnect.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
